src/plots/layout_attributes.js

```diff
@@ -292,6 +292,19 @@ module.exports = {
             'Sets the background color of the plotting area in-between x and y axes.'
         ].join(' ')
     },
+    autotypenumbers: {
+        valType: 'enumerated',
+        values: ['convert types', 'strict'],
+        dflt: 'convert types',
+        role: 'info',
+        editType: 'calc',
+        description: [
+            'Using *strict* a numeric string in trace data is not converted to a number.',
+            'Using *accept strings* a numeric string in trace data may be',
```

Comment:
```suggestion
            'Using *convert types* a numeric string in trace data may be',
```
Unless we want to go back to `'accept strings'` since this isn't being used for dates. I still think there are cases where it would make a difference for dates, but it's definitely a very unlikely situation - much harder to come upon accidentally than I had initially thought - so I'm OK with ignoring it. Maybe though let's keep `'convert types'` since it gives us the flexibility to later address this date case without adding a new value here, if it does become important to someone.

Reply:
Good eye! Done in https://github.com/plotly/plotly.js/pull/5240/commits/4b5bfcc73782f935210db720fc52accde634869e.